lib/trmnl/i18n/locales/web_ui/de-AT.yml

```
@@ -198,8 +198,8 @@ de-AT:
       special_function: Sonderfunktion
       special_function_hint: Richte eine Doppelklickfunktion für die Taste auf der Rückseite deines Geräts ein.
       special_function_learn_more: Erfahre mehr zu den Sonderfunktionen __hier__.
-      temperature_profile: Temperature Profile
-      temperature_profile_hint: Don't change unless instructed. For displays with washed out images.
+      temperature_profile: Temperaturprofil
+      temperature_profile_hint: Nicht ändern, außer es wird ausdrücklich angeordnet. Für Displays mit ausgewaschenen Bildern.
```

```suggestion
      temperature_profile_hint: Nur ändern, wenn du dazu aufgefordert wirst. Für Displays mit zu blasser Darstellung.
```
I think this sounds less harsh, and I wouldn't say e-ink can look "ausgewaschen".